### PR TITLE
Update donate cta in 102 What's New page

### PR DESCRIPTION
**Pull request by micahilbery**
Screenshot from 2022-08-15 13-40-57

### CODE REVIEW

**Comment by micahilbery** [comment]
I would also like to remove the `site_header_nav` on this page only but I haven't found a way to do that without giving it a custom `base-resp` if anyone has any tips on how I could best go about doing that.

**Comment by Alecaddd** [comment]
@micahilbery check this: https://github.com/thundernest/thunderbird-website/blob/db224551f77a09303c5b17533faa4d68cd5d6dfb/website/_includes/base-resp.html#L16

We already have some inline condition to check the type of page.
You can print out `active_page` and see what it returns and use that as condition to exclude the main nav.

Also, remember to change the UTM link in the donate button.

**Comment by micahilbery** [comment]
image

**Comment by ryanleesipes** [comment]
The UTM link should be: https://give.thunderbird.net?utm_source=whats_new_page&utm_medium=thunderbird&utm_campaign=whats_new_102&utm_content=cta

(this is for where the donate button goes).

**Review by ryanleesipes — CHANGES_REQUESTED** [review]
Please add the following link to the donate button before I can approve: 

https://give.thunderbird.net?utm_source=whats_new_page&utm_medium=thunderbird&utm_campaign=whats_new_102&utm_content=cta

Also, @Alecaddd discussed potentially adjusting the donate box color. If he says we should do that, I think that might help the content stand out more. Otherwise the design looks good to me.

**Comment by micahilbery** [comment]
> Please add the following link to the donate button before I can approve:
> 
> https://give.thunderbird.net?utm_source=whats_new_page&utm_medium=thunderbird&utm_campaign=whats_new_102&utm_content=cta

Currently our donate link helper function doesn't support creating a link like this. That function will need to be updated before this can happen or we will have to use a direct link which might interfere somewhere else I'm not sure.

File: website/thunderbird/102.0/whatsnew/index.html
Comment (Alecaddd) [inline website/thunderbird/102.0/whatsnew/index.html:28]: I think we should make it a tad smaller, by re-enabling the `max-w-2xl` and using that.

**Review by Alecaddd — CHANGES_REQUESTED** [review]
Thanks for working on this so quickly.
This needs to be fixed for mobile, as the donation appeal should be center aligned when the image disappear.

**Comment by ryanleesipes** [comment]
> > Please add the following link to the donate button before I can approve:
> > 
> > https://give.thunderbird.net?utm_source=whats_new_page&utm_medium=thunderbird&utm_campaign=whats_new_102&utm_content=cta
> 
> Currently our donate link helper function doesn't support creating a link like this. That function will need to be updated before this can happen or we will have to use a direct link which might interfere somewhere else I'm not sure.
> 
> 

@Sancus can you help resolve this?

**Review by Alecaddd — APPROVED** [review]
This looks good for me, thanks.
As usual, @Sancus should be the final reviewer and the one to pull the trigger on merging this.

**Review by ryanleesipes — APPROVED** [review]

**Comment by ryanleesipes** [comment]
Has my approval too. @Sancus can you do a quick review and, if it looks good, get it merged?

**Pull request merged**
